final_task/build/lib/rss_parser/Output_functions.py

```diff
@@ -0,0 +1,332 @@
+    3. Some problems can occur if there are no alternative text, solving them with changing list of alts
+    :param main_source:
+    :param the_feed:
+    :param num_of_news:
+    :return:
+    """
+    if getting_num_of_news(the_feed, num_of_news) > len(the_feed.entries):
+        print("You want to get more news than exist.")
+        print("Printing all news from the source!")
+
+    print("------------------------")
+    print("Source: ", printing_title(the_feed))
+    pack_of_news = []
+    pack_of_images_links = getting_images_links(the_feed, num_of_news)
+    pack_of_alts = getting_alt_text(the_feed, num_of_news)
+    corrected_pack_of_alts = [el for el, _ in groupby(pack_of_alts)]
+
+    for num, item in enumerate(the_feed.entries[:num_of_news]):
+        pack_of_news.append(getting_novelty(item, num, pack_of_images_links, corrected_pack_of_alts, main_source))
+    return pack_of_news
+
+
+def getting_novelty(item, number, corrected_pack_of_images_links, corrected_pack_of_alts, main_source):
+    """
+    It was really difficult to get images because of variety of templates how sites leave link for image
+    Here we create object of Novelty class and fill it with our title, description and etc.
+    Then if some problems with images or alt.text occur we use
+    :param main_source:
+    :param item:
+    :param number:
+    :param corrected_pack_of_images_links:
+    :param corrected_pack_of_alts:
+    :return:
+    """
+
+    try:
+        novelty = Novelty(number + 1, item.get('title', ''),
+                          item.get('published', ''),
+                          item.get('link', ''),
+                          clean_html(item.get('description', '')),
+                          corrected_pack_of_images_links[number],
+                          corrected_pack_of_alts[number],
+                          getting_corrected_time(item),
+                          main_source)
+    except IndexError:
+        novelty = Novelty(number + 1, item.get('title', ''),
+                          item.get('published', ''),
+                          item.get('link', ''),
+                          clean_html(item.get('description', '')),
+                          corrected_pack_of_images_links[number],
+                          'No alternative text.',
+                          getting_corrected_time(item),
+                          main_source)
+    return novelty
+
+
+def getting_full_info(pack_of_news):
+    """
+    Getting full info from news
+    try-except for printing links and alternative text
+    :param pack_of_news:
+    :return:
+    """
+    for novelty in pack_of_news:
+        print()
+        print("{0}.".format(novelty.number_of_novelty), "Title: ", novelty.title_of_novelty)
+        print("Published: ", novelty.time_of_novelty)
+        print("Link: ", novelty.source_link)
+        print("Description: ")
+        print(pprint.pformat(novelty.description, width=115))
+        try:
+            print()
```

Comment:
There is a better solution if you need to print an empty line instead of `print()`